src/main/res/values/strings.xml

```diff
@@ -283,21 +283,21 @@
     <string name="comment_edit_done">Comment edited. Changes will appear after you refresh.</string>
     <string name="post_submit_title_hint">Title</string>
     <string name="post_submit_subreddit_hint">Subreddit</string>
-    <string name="post_captcha_title">Captcha</string>
+    <string name="post_captcha_title">CAPTCHA</string>
     <string name="post_captcha_submit_button">Submit</string>
     <string name="post_submit_done">Post submitted.</string>
 
     <!-- Bug Reporting -->
 
     <string name="bug_title">An error has occurred.</string>
-    <string name="bug_message">RedReader has encountered an error.\n\nWould you like to send a bug report to the developer? If so, an email will be filled in for you with all the relevant information, and you can inspect the contents before sending.</string>
-    <string name="bug_button_send">Send bug report by email</string>
+    <string name="bug_message">RedReader has encountered an error.\n\nWould you like to send a bug report? An inspectable e-mail will be filled in for you with all the relevant information for you to send.</string>
+    <string name="bug_button_send">Send bug report per e-mail</string>
```

Comment:
"email" is better imo (applies to all changes of this type)

Reply:
Errors are invariably added in translation with email, because other languages require the dash, whereas English allows both.